helpers/logging/middleware.go

```diff
@@ -36,8 +54,17 @@ func AddLoggingHandler(next http.Handler) http.Handler {
 		reqId, _ := uuid.Uuid()
 		reqId = fmt.Sprintf("[req-id %s]", reqId)
 		l.Println(reqId, r.Method, r.URL)
-		lr := &logResponseWriter{w, 0}
+		lr := &logResponseWriter{w, 0, nil}
+		br := &logRequestReader{r.Body, make([]byte, r.ContentLength)}
+		r.Body = br
 		next.ServeHTTP(lr, r)
+
+		reqstr := fmt.Sprintf("%s", br.Body)
```

Comment:
yeah, i can adjust it so that it only does those if debug is enabled. the Sprintf's are only needed to massage the `[]byte` into a `string`.